src/scaffolding/master.scaffolding.hbs

```diff
@@ -59,6 +59,59 @@
 	{{bioInclude 'components/BioScrollAnimation/scaffolding/BioScrollAnimation' }}
 </section>
 
+<section class="resources">
+        <div class="row row--reducedWidth">
+            <div class="col">
+                <h3 class="resources__headline--blue">Code & Resources</h3>
+                <bio-resource logo-baseline image-border-color="blue" data-resources="[{paths: ['components/BioResource/index.js']}]" imgsrc="_assets/github-logo.svg">
+                    <p>See the Bioto.pe code on GitHub</p>
+                    <div class="buttonWrapper">
+                        <bio-button title="Github" modifier="blue" data-resources="[{paths: ['components/BioButton/index.js']}]"></bio-button>
```

Comment:
You don't need to provide the `bio-button`s with the `data-resource` if you use the [dependency](https://element.biotope.sh/#/advanced?id=dependencies) feature of bio-element.